index.js

```diff
@@ -192,22 +192,34 @@ module.exports = class Hyperdrive extends ReadyResource {
   async clear (name, { storageInfo = false } = {}) {
     const node = await this.entry(name)
     if (node === null) return 0
-
-    await this.getBlobs()
+    if (!this.blobs) return 0 // if no blobs, nothing to clear
```

Comment:
I think this generates a bug where the test passes because it was expecting zero, will see to adjust the test